sqldelight-gradle-plugin/src/test/fixtures/bind-object/expected/com/sample/TestModel.java

```diff
@@ -151,4 +130,15 @@ public void some_delete(SQLiteProgram program, Object arg1) {
       return new Mapper<T>(this);
     }
   }
+
+  final class Some_deleteStatement {
+    public final SQLiteProgram program;
+
+    Some_deleteStatement(SQLiteDatabase database) {
+      program = database.compileStatement(""
+              + "WITH rubbish AS (VALUES (?))\n"
```

Comment:
Hmm that seems broken. It should render all those lines with +4. Are you saying JavaPoet is doing the secondary line wrapping and automatically adding the extra double indent?